rules.d/90-iocost.rules.in

```diff
@@ -0,0 +1,12 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+ACTION=="add", SUBSYSTEM=="block", \
+    IMPORT{builtin}="hwdb 'block:*:name:$env{ID_MODEL}:'", \
+    RUN+="iocost apply $env{DEVNAME}"
```

Comment:
so this rule will run for all block devices now, spawning an external tool for each. On large disk arrays this will be expensive. Which is fine — if there actually was iocost data set. But typically there will not be. Hence, please conditionalize this out if no iocost props are set. for example if IOCOST_SOLUTIONS is not set